src/main/target/MATEKF405SE/target.h

```diff
@@ -75,13 +75,16 @@
 #define USE_MAG_LIS3MDL
 
 #define USE_RANGEFINDER
+#define USE_RANGEFINDER_MSP
 #define USE_RANGEFINDER_HCSR04_I2C
 #define RANGEFINDER_I2C_BUS     BUS_I2C2
 
 #define PITOT_I2C_BUS           BUS_I2C2
 
 #define TEMPERATURE_I2C_BUS     BUS_I2C2
 
+#define USE_OPTICAL_FLOW
+#define USE_OPFLOW_MSP
```

Comment:
`USE_OPFLOW_MSP` is already defined for all targets having more than 256k flash, it is not needed.